plutus-tx-plugin/test/Blueprint/Tests.hs

```diff
@@ -39,35 +51,42 @@ contractBlueprint =
             preambleLicense = Just "MIT"
           },
       contractValidators =
-        Set.singleton
-          MkValidatorBlueprint
-            { validatorTitle = "Acme Validator",
-              validatorDescription = Just "A validator that does something awesome",
-              validatorParameters =
-                Just $
-                  pure
-                    MkParameterBlueprint
-                      { parameterTitle = Just "Acme Parameter",
-                        parameterDescription = Just "A parameter that does something awesome",
-                        parameterPurpose = Set.singleton Purpose.Spend,
-                        parameterSchema = definitionRef @Fixture.Params
-                      },
-              validatorRedeemer =
-                MkArgumentBlueprint
-                  { argumentTitle = Just "Acme Redeemer",
-                    argumentDescription = Just "A redeemer that does something awesome",
-                    argumentPurpose = Set.fromList [Purpose.Spend, Purpose.Mint],
-                    argumentSchema = definitionRef @Fixture.Redeemer
-                  },
-              validatorDatum =
-                Just
-                  MkArgumentBlueprint
-                    { argumentTitle = Just "Acme Datum",
-                      argumentDescription = Just "A datum that contains something awesome",
-                      argumentPurpose = Set.singleton Purpose.Spend,
-                      argumentSchema = definitionRef @Fixture.Datum
-                    },
-              validatorCompiledCode = Just Fixture.serialisedScript
-            },
+        Set.fromList
+          [ MkValidatorBlueprint
+              { validatorTitle =
+                  "Acme Validator #1",
+                validatorDescription =
+                  Just "A validator that does something awesome",
+                validatorParameters =
+                  Just . pure $
+                    $(deriveParameterBlueprint ''Fixture.Params (Set.singleton Purpose.Spend)),
+                validatorRedeemer =
+                  $(deriveArgumentBlueprint ''Fixture.Redeemer (Set.singleton Purpose.Spend)),
+                validatorDatum =
+                  Just $ $(deriveArgumentBlueprint ''Fixture.Datum (Set.singleton Purpose.Spend)),
+                validatorCompiledCode =
+                  Just $ Fixture.serialisedScript Fixture.validatorScript1
+              },
+            MkValidatorBlueprint
+              { validatorTitle =
+                  "Acme Validator #2",
+                validatorDescription =
+                  Just "Another validator that does something awesome",
+                validatorParameters =
+                  Just $
+                    $(deriveParameterBlueprint ''Fixture.Param2a (Set.singleton Purpose.Mint))
+                      :| [ $( deriveParameterBlueprint
```

Comment:
Wait, is this a non-empty list? That seems wrong?

Reply:
Fixed